src/Media.Plugin.Abstractions/MediaStoreOptions.cs

```diff
@@ -208,6 +208,10 @@ public int CompressionQuality
             }
         }
 
+        /// <summary>
+        /// Store provided location
+        /// </summary>
+        public Location UseLocation { get; set; }
```

Comment:
I don't think this is the correct name. Shouldn't it just be "Location".

UseLocation makes it sound like a boolean